### PR TITLE
Minor issue in htmlredirect.go - reservedNames in Windows - COM0 and LPT0 are also invalid on Windows 7

### LINKED ISSUE #2883: windows:  COM0 and LPT0 are also invalid on Windows 7?
I just tried creating files named COM0 as well as LPT0 on my machine and they both give the error "The specified device name is invalid" (the same as the others).

See: htmlredirect.go
Line: 73 (reservedNames := []string{"CON", "PRN", "AUX", "NUL", "COM1", "COM2", "COM3", "COM4", "COM5", "COM6", "COM7", "COM8", "COM9", "LPT1", "LPT2", "LPT3", "LPT4", "LPT5", "LPT6", "LPT7", "LPT8", "LPT9"})

Tested on Windows 7 Home Premium x64 SP1 (English).

I could try on Windows XP and Windows 10 if needed.

### PR DESCRIPTION
**Pull request by magikstm**
Fixes #2883

### CODE REVIEW

**Comment by bep** [comment]
Please add a more descriptive issue title than "Additions for #2883". It is impossible to understand when browsing the issues.

Also look at the commit message guidelines in the README.

**Comment by magikstm** [comment]
Thanks. I reviewed it.

It's my first commit on github. I've only used it at work before.

File: target/htmlredirect.go [hunk 1]
```diff
@@ -70,7 +70,7 @@ func (h *HTMLRedirectAlias) Translate(alias string) (aliasPath string, err error
 	// See "Naming Files, Paths, and Namespaces" on MSDN
 	// https://msdn.microsoft.com/en-us/library/aa365247%28v=VS.85%29.aspx?f=255&MSPPError=-2147217396
 	msgs := []string{}
-	reservedNames := []string{"CON", "PRN", "AUX", "NUL", "COM1", "COM2", "COM3", "COM4", "COM5", "COM6", "COM7", "COM8", "COM9", "LPT1", "LPT2", "LPT3", "LPT4", "LPT5", "LPT6", "LPT7", "LPT8", "LPT9"}
+	reservedNames := []string{"CON", "PRN", "AUX", "NUL", "COM0", "COM1", "COM2", "COM3", "COM4", "COM5", "COM6", "COM7", "COM8", "COM9", "LPT0", "LPT1", "LPT2", "LPT3", "LPT4", "LPT5", "LPT6", "LPT7", "LPT8", "LPT9"}
```
Comment (moorereason) [inline target/htmlredirect.go]: This is not the right solution.  I'll comment more on the original issue.

Comment (moorereason) [inline target/htmlredirect.go]: I don't think I was awake when I commented on this.  Disregard.

**Comment by anthonyfok** [comment]
Merged as 9c19ef0f.  Thank you for your contribution!

**Pull request closed without merging**